frontend/package.json

```diff
@@ -2,71 +2,80 @@
     "name": "frontend",
     "version": "0.1.0",
     "private": true,
+    "type": "module",
     "dependencies": {
-        "@aldabil/react-scheduler": "^2.7.26",
-        "@aws-amplify/auth": "^5.6.4",
+        "@aldabil/react-scheduler": "^2.8.8",
```

Comment:
Please revert the upgrade to react-scheduler. He introduced a breaking change to the way timezone overriding works in 2.8 and I haven't had time to refactor the Dojo code to work with that.